rebar.config.script

```diff
@@ -15,6 +15,18 @@
 ConfigureEnv = filename:join(filename:dirname(SCRIPT), "config.erl"),
 os:putenv("COUCHDB_CONFIG", ConfigureEnv).
 
+OTP_VSN = erlang:system_info(compat_rel),
+
+Features = lists:filter(fun({_K, Flag}) -> Flag end, [
+    {'HAS_CALLBACKS', OTP_VSN > 14}
+]),
+
+Properties = Features ++ [
+    {'OTP_VSN', OTP_VSN},
+    {'OTP_RELEASE', erlang:system_info(otp_release)}
+],
```

Comment:
further hm, how would we use ?OTP_RELEASE in code? We couldn't use a macro conditional on its value in couch_crypto as far as I can see.